examples/task-tracker/client/tasks/add_comment_to_task_parameters.go

```diff
@@ -112,6 +133,16 @@ func (o *AddCommentToTaskParams) SetID(id int64) {
 	o.ID = id
 }
 
+// Validate these params
+func (o *AddCommentToTaskParams) Validate(formats strfmt.Registry) error {
+
+	if err := o.Body.Validate(formats); err != nil {
```

Comment:
Ok weird, I'll try to track that issue later today. Can you please also check if my conditionals added in the template are reasonable, as you might have a better idea whether or not those will hold up for all permutations.